iredis/client.py

```diff
@@ -80,7 +80,9 @@ def get_server_info(self):
         # safe to decode Redis's INFO response
         info_resp = utils.ensure_str(self.connection.read_response())
 
-        version = re.findall(r"^redis_version:([\d\.]+)\r\n", info_resp, re.MULTILINE)[0]
+        version = re.findall(r"^redis_version:([\d\.]+)\r\n", info_resp, re.MULTILINE)[
+            0
```

Comment:
This place is not particularly well formatted.

Reply:
Black's uncompromising... I can accept that.